dev/NavigationView/NavigationView.cpp

```diff
@@ -2970,7 +2970,9 @@ void NavigationView::UpdateHeaderVisibility()
 
 void NavigationView::UpdateHeaderVisibility(winrt::NavigationViewDisplayMode displayMode)
 {
-    bool showHeader = AlwaysShowHeader() || displayMode == winrt::NavigationViewDisplayMode::Minimal;
+    // Ignore AlwaysShowHeader property in case DisplayMode is minimum and we're not a Top NavigationView
```

Comment:
```suggestion
    // Ignore AlwaysShowHeader property in case DisplayMode is Minimal and it's not Top NavigationView
```

Reply:
Fix looks good to me.

Reply:
@YuliKl  Can you confirm if the fix is reasonable? Without the fix, In top navigationview, it always show header not matter AlwaysShowHeader is true or false.

Reply:
Yes, this looks good. Top NavigationView should respect the AlwaysShowHeader property.